axelor-supplychain/src/main/resources/views/SaleOrderLine.xml

```diff
@@ -8,19 +8,134 @@
+          <field name="analyticDistributionTemplate"
+            grid-view="analytic-distribution-template-grid"
+            form-view="analytic-distribution-template-form" canEdit="false"
+            onChange="action-sale-order-line-group-analytic-distribution-template-onchange"
+            onSelect="action-sale-order-line-attrs-set-domain-analytic-distribution-template"/>
+          <field name="axis1AnalyticAccount" grid-view="analytic-account-grid"
+            form-view="analytic-account-form" readonlyIf="analyticDistributionTemplate"
+            onSelect="action-sale-order-line-method-set-axis-domain"
+            onChange="action-sale-order-line-method-axis-create-analytic-line"/>
+          <panel name="axisAnalyticAccountPanel" colSpan="12"
+            readonlyIf="analyticDistributionTemplate">
+            <field name="axis2AnalyticAccount" grid-view="analytic-account-grid"
+              form-view="analytic-account-form" readonlyIf="analyticDistributionTemplate != null"
+              onSelect="action-sale-order-line-method-set-axis-domain"
+              onChange="action-sale-order-line-method-axis-create-analytic-line"/>
+            <field name="axis3AnalyticAccount" grid-view="analytic-account-grid"
+              form-view="analytic-account-form" readonlyIf="analyticDistributionTemplate != null"
+              onSelect="action-sale-order-line-method-set-axis-domain"
+              onChange="action-sale-order-line-method-axis-create-analytic-line"/>
+            <field name="axis4AnalyticAccount" grid-view="analytic-account-grid"
+              form-view="analytic-account-form" readonlyIf="analyticDistributionTemplate != null"
+              onSelect="action-sale-order-line-method-set-axis-domain"
+              onChange="action-sale-order-line-method-axis-create-analytic-line"/>
+            <field name="axis5AnalyticAccount" grid-view="analytic-account-grid"
+              form-view="analytic-account-form" readonlyIf="analyticDistributionTemplate != null"
+              onSelect="action-sale-order-line-method-set-axis-domain"
+              onChange="action-sale-order-line-method-axis-create-analytic-line"/>
+          </panel>
+          <panel-related name="analyticMoveLineListPanel" field="analyticMoveLineList"
+            grid-view="analytic-move-line-distribution-grid"
+            form-view="analytic-move-line-distribution-form" colSpan="12"
+            onChange="action-sale-order-line-group-analytic-move-line-list-onchange"/>
+          <field name="$isValidAnalyticMoveLineList" type="boolean" hidden="true"/>
+        </panel>
+      </insert>
+
+      <insert position="after">
+        <panel name="settingsPanel" readonlyIf="$isReadOnly !== false" title="Configuration">
+          <field name="saleSupplySelect"
+            onChange="action-group-sale-saleorderline-salesupplyselect-onchange"
+            if="__config__.app.isApp('supplychain')"/>
```

Comment:
Can you check if this is actually needed to check supplychain app in supplychain module ?